x-pack/plugin/esql/qa/testFixtures/src/main/resources/knn-function.csv-spec

```diff
@@ -227,62 +227,76 @@ azure       | [240.0, 255.0, 255.0]
 ;
 
 knnWithNonPushableConjunction
-required_capability: knn_function_v3
+required_capability: knn_function_v4
 
 from colors metadata _score
 | eval composed_name = locate(color, " ") > 0 
-| where knn(rgb_vector, [128,128,0], 140) and composed_name == false
+| where knn(rgb_vector, [128,128,0], 10) and composed_name == false
```

Comment:
We can see the change in action - we no longer need to use a large number for k to maintain semantics, nor to use limit at the end.